pyproject.toml

```diff
@@ -49,7 +49,7 @@ optional-dependencies.dev = {file = ["requirements-dev.txt"]}
 where = ["src"]
 
 [tool.setuptools.package-data]
-dinglehopper = ["templates/*"]
+dinglehopper = ["templates/*", "*.json"]
```

Comment:
Indeed, it is currently missing!

See [here](https://github.com/bertsky/ocrd_detectron2/blob/master/MANIFEST.in) for a MANIFEST.in that ensures this is included (independent of the setuptools discovery goo).

Reply:
I checked an non-editable install with master, setuptools-ocrd already takes care of this → Inclusion in `tools.setuptools.package-data` is redundant and `MANIFEST.in` would make it super-redundant ;) 

But it also doesn't here to include it in `tools.setuptools.package-data`, so this can be left in here.

Reply:
(And and the OCR-D CLI actually reads this. A proper (non-editable) install may have been broken before, but I'm too lazy to check.)